fix: API v2, wording, style (#486)
BoltzExchange/boltz-backend: docs/lifecycle.md

   * `invoice.pending`: if paying the invoice is in progress
   * `invoice.paid`: if paying the invoice was successful or
   * `invoice.failedToPay`: if paying the invoice failed. In this case the user needs to broadcast a refund transaction to reclaim the locked up chain bitcoin.

> 6. `transaction.claim.pending`: This status indicates that Boltz is ready for the creation of a cooperative signature for a keypath spend. Taproot Swaps are not claimed immediately by Boltz after the invoice has been paid, but instead Boltz waits for the API client to post a signature for a key path spend, which is the more efficient and private method. If the API client does not cooperate in a key path spend, Boltz includes its claim transaction in a script path spend batch.

Reviewer: ```suggestion
6. `transaction.claim.pending`: This status indicates that Boltz is ready for the creation of a cooperative signature for a keypath spend. Taproot Swaps are not claimed immediately by Boltz after the invoice has been paid, but instead Boltz waits for the API client to post a signature for a key path spend. If the API client does not cooperate in a key path spend, Boltz will eventually claim via the script path.
```

The reason is not really relevant for the status and the batching is also not important to the client
